bigquery/README.md

```diff
@@ -9,6 +9,7 @@ within docker.
 * If you want to use a [sample public dataset](https://bigquery.cloud.google.com/table/google.com:biggene:platinum_genomes.sample_info):
   * [Install bq](https://cloud.google.com/bigquery/docs/bq-command-line-tool#installation)
 if you haven't done so already.
+  * [Install virtualenv](https://pypi.python.org/pypi/virtualenv).
```

Comment:
Can you move this to "Run indexer" step and change to "Install virtualenv if you haven't done so already."